src/backend/database/migrations/2025_02_26_233553_create_vodacom_mobile_money_tables.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Let's add a status column here, that could be an enum as ['pending','succeeded','failed'] or [0,1,-1]